### PR TITLE
Securonix detailed desc

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by Itay4** [comment]
lets also add it to the integration doc

File: Packs/Securonix/Integrations/Securonix/README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Use the Securonix integration to manage incidents and watchlists.
 Integration was build and tested with SNYPR Version 6.3.
```
Comment (Itay4) [inline Packs/Securonix/Integrations/Securonix/README.md:3]: do we know if this is the version for both cloud and on-prem ?

Comment (bakatzir) [reply Packs/Securonix/Integrations/Securonix/README.md:3]: No, as we get access to  cloud base only. I know for a fact that it is working for an on prem of version 6.2 as well.

Comment (bakatzir) [reply Packs/Securonix/Integrations/Securonix/README.md:3]: according to the vendor, API should be compatible for both deployments.

File: Packs/Securonix/Integrations/Securonix/README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 ## Overview
 Use the Securonix integration to manage incidents and watchlists.
-Integration was build and tested with SNYPR Version 6.3.
+Integration was build and tested with SNYPR Versions: 6.2, 6.3.
```
Comment (amshamah419) [inline Packs/Securonix/Integrations/Securonix/README.md:3]: Integration was built and tested with SNYPR Versions: 6.2, 6.3.

**Pull request merged**